src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/TelemetryReportingLSPRequestInvoker.cs

```diff
@@ -78,12 +80,16 @@ public TelemetryReportingLSPRequestInvoker(LSPRequestInvoker requestInvoker, ITe
         }
     }
 
-    private IDisposable? Track(string name, string method, string languageServerName)
+    internal IDisposable? Track(string name, string method, string languageServerName, Guid correlationId = default(Guid))
     {
+        if (correlationId == Guid.Empty)
+            return default;
```

Comment:
Looks like this answers my previous comment: Every methoid in this class is now calling `Track`, which is essentially a no-op, so this class is not really adding any value at all and can be removed.